Tools/Scripts/webkitpy/browserperfdash/browserperfdash_runner.py

```diff
@@ -78,6 +80,10 @@ def __init__(self, args):
                              'test_id': None,
                              'test_version': None,
                              'test_data': None}
+        if args.timestamp:
+            self._result_data['timestamp'] = int(args.timestamp)
```

Comment:
Casting to int might not be necessary since `type=int` in parser.

Reply:
Right, it is a bit redundant. Will remove it.
Thanks for the review, btw